start.go

```diff
@@ -34,18 +40,33 @@ func start(c *cli.Context) error {
 	if c.String("log-level") != "" {
 		denvs = append(denvs, "GIN_MODE="+c.String("log-level"))
 	}
-	// Socket mount: docker run --rm -it --name functions -v ${PWD}/data:/app/data -v /var/run/docker.sock:/var/run/docker.sock -p 8080:8080 fnproject/functions
-	// OR dind: docker run --rm -it --name functions -v ${PWD}/data:/app/data --privileged -p 8080:8080 fnproject/functions
+
+	// docker-in-docker or socker bind mount?
+	isDind := true
+	if c.String("docker-mode") == "docker-in-docker" && runtime.GOOS == "windows" {
```

Comment:
this is not only windows-specific, on Mac i don't really rely on dind mode rather than running alpine image with fn binary inside and have my standalone docker.